Utils/helper_functions.py

```diff
@@ -0,0 +1,60 @@
+# functions to find files, get median,
+
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+
+# function to open a file and return the data in a pandas dataframe
+def open_file(file_path):
+    df = pd.read_csv(file_path, sep=",")
+    return df
+
+
+# function to find the median row of a column in a dataframe and return the value and index of the row
+def find_median(df, column):
+    median = df[column].median()
+    median_index = df[column].sub(median).abs().idxmin()
+    print(median_index)
```

Comment:
If the intent with this print is to help observe behavior within this function, consider the use of the [`logging`](https://docs.python.org/3/howto/logging.html) module to assist with logging operations.